app/src/main/res/layout/content_no_tickets.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="@dimen/layout_margin_extra_large"
+    android:orientation="vertical">
+
+    <RelativeLayout
+        android:layout_width="@dimen/item_image_view_large"
+        android:layout_height="@dimen/item_image_view_large"
+        android:layout_gravity="center_horizontal"
+        android:layout_marginTop="@dimen/layout_margin_large">
+
+        <ImageView
+            android:layout_width="@dimen/item_image_view_large"
+            android:layout_height="@dimen/item_image_view_large"
+            android:layout_centerInParent="true"
+            android:scaleType="centerCrop"
+            app:srcCompat="@drawable/round_shape" />
+
+        <ImageView
+            android:layout_width="@dimen/item_image_view"
+            android:layout_height="@dimen/item_image_view"
+            android:tint="@color/colorPrimary"
```

Comment:
This will fail on pre lollipop devices

Reply:
> This will fail on pre lollipop devices

Should I use `android:indeterminateTint` instead of `android:tint` attribute?

Reply:
> This will fail on pre lollipop devices

Updated with `AppCompatImageView`

Reply:
@liveHarshit I guess the problem is with the `tint` attribute.

Reply:
Review comments should be inferred for the line of code immediately above them

Reply:
Then how can I tint, should I tint programmatically as `wrapDrawable` or something else. Please suggest me?

Reply:
Then how can I tint, should I tint programmatically as `wrapDrawable` or something else. Please suggest me?

Reply:
> Review comments should be inferred for the line of code immediately above them

Then how can I tint, should I tint programmatically as `wrapDrawable` or something else. Please suggest me?

Reply:
> Review comments should be inferred for the line of code immediately above them

Then how can I tint, should I tint programmatically as `wrapDrawable` or something else. Please suggest me?

Reply:
> Review comments should be inferred for the line of code immediately above them

Then how can I tint, should I tint programmatically as `wrapDrawable` or something else. Please suggest me?

Reply:
> Review comments should be inferred for the line of code immediately above them

Then how can I tint, should I tint programmatically as `wrapDrawable` or something else. Please suggest me?

Reply:
> Review comments should be inferred for the line of code immediately above them

Then how can I tint, should I tint programmatically as `wrapDrawable` or something else. Please suggest me?

Reply:
> Review comments should be inferred for the line of code immediately above them

Then how can I tint, should I tint programmatically as `wrapDrawable` or something else. Please suggest?